experiment/run_experiment.py

```diff
@@ -185,6 +185,8 @@ def read_and_validate_experiment_config(config_filename: str) -> Dict:
             Requirement(False, str, True, ''),
         'runner_num_cpu_cores':
             Requirement(False, int, False, ''),
+        'seed_sampling':
+            Requirement(False, dict, False, ''),
```

Comment:
I guess we only expect the `seed_sampling` dictionary to define two items in `experiment_config.yaml`:
1. `distribution` (a str, either `'UNIFORM'` or `'EXP'`).
1. `mean_seed_utilization` (a float, within `[0,1]]`).

Would it be better to define them as two individual items in the YAML file and validate them here?
This allows us to capture any mistakes of them (e.g., `'Unimplemented sampling algorithm'`) immediately when the experiment launches, before waiting for a long time to build benchmarks & fuzzers.

Reply:
Yeah, I don't have any strong preferences for the YAML file, this sounds reasonable to me

Reply:
Thanks!

Reply:
Configuration is now flattened out, as below:
[example-config.yaml.txt](https://github.com/google/fuzzbench/files/11397420/example-config.yaml.txt)

Reply:
Looks amazing, thanks!